lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -198,7 +198,8 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
               <.live_component
                 module={LivebookWeb.Hub.Teams.DeploymentGroupListComponent}
                 id="hub-deployment-groups-list"
-                hub_id={@hub.id}
+                hub={@hub}
+                deployment_group={@deployment_group}
```

Comment:
This is why `@deployment_group.id` was failing in weird ways. You can remove this line:

```suggestion
```

Reply:
I cannot, if I do, I get an error:

`key :deployment_group not found in: %{
  id: "hub-deployment-groups-list",
  socket: #Phoenix.LiveView.Socket<
    id: "phx-F8BDYY5ADMsTxAZj",
    endpoint: LivebookWeb.Endpoint,
    view: LivebookWeb.Hub.EditLive,
    parent_pid: nil,
    root_pid: nil,
    router: LivebookWeb.Router,
    assigns: #Phoenix.LiveView.Socket.AssignsNotInSocket<>,
    transport_pid: nil,
    ...
  >,`